website/docs/reference/dbt-jinja-functions/execute.md

```diff
@@ -40,7 +40,7 @@ Compilation Error in model order_payment_methods (models/order_payment_methods.s
   'None' has no attribute 'table'
 
 ```
-This is because Line #11 assumes that a <Term id="table" /> has been returned, when, during the parse phase, this query hasn't been run.
+This is because Line #11 (`{% set payment_methods = results.columns[0].values() %}`) assumes that a <Term id="table" /> has been returned, when, during the parse phase, this query hasn't been run.
```

Comment:
```suggestion
This is because line #11 in the earlier code example (`{% set payment_methods = results.columns[0].values() %}`) assumes that a <Term id="table" /> has been returned, when, during the parse phase, this query hasn't been run.
```